routes/image_settings.py

```diff
@@ -2,6 +2,59 @@
 import requests
 from flask import render_template
 
+@app.route("/set55")
+def set_55():
+    payload = {
+        "name": "camera.setOptions",
+        "parameters": {
+            "options": {
+                "fileFormat": {
+                    "type": "jpeg",
+                    "width": 5504,
+                    "height": 2752,
+                }
+            }
+        },
+    }
+    response = requests.post(base_url + "commands/execute", json=payload)
+    return render_template(
+        "image_settings.html", response=response.text, title="set image resolution to 5.5K",
+        active_nav="set55"
+    )
+
+@app.route("/set11")
+def set_11():
+    payload = {
+        "name": "camera.setOptions",
+        "parameters": {
+            "options": {
+                "fileFormat": {
+                    "type": "jpeg",
+                    "width": 11008,
+                    "height": 5504,
+                }
+            }
+        },
+    }
+    response = requests.post(base_url + "commands/execute", json=payload)
+    return render_template(
+        "image_settings.html", response=response.text, title="set image resolution to 11K",
+        active_nav="set11"
+    )
+
+@app.route("/nodelay")
+def nodelay():
+    payload = {
+        "name": "camera._stopSelfTimer",
```

Comment:
Use exposureDelay and set value to 0.  https://github.com/ricohapi/theta-api-specs/blob/main/theta-web-api-v2.1/options/exposure_delay.md

![image](https://github.com/codetricity/theta-flask-desk/assets/5799860/cf4a1745-d550-46c7-a4da-a10565b420a1)